src/operations/find_and_modify.ts

```diff
@@ -10,6 +10,15 @@ import { Sort, SortForCmd, formatSort } from '../sort';
 import type { ClientSession } from '../sessions';
 import type { WriteConcern, WriteConcernSettings } from '../write_concern';
 
+/** @public */
+export const ReturnDocument = Object.freeze({
+  BEFORE: 'before',
+  AFTER: 'after'
```

Comment:
`ReturnDocument.before` and `ReturnDocument.after` would be my preference here, seems just 🤏  that much easier to type. Not a strong opinion though

Reply:
I checked and we currently have about a 50-50 split on uppercase vs camelcase in existing enums; my personal preference is to use uppercase with enums to emphasize the static nature of the thing being referenced as opposed to just being a regular property on a regular object

Reply:
SGTM